Firmware/ultralcd.cpp

```diff
@@ -3265,6 +3263,11 @@ uint8_t lcd_show_multiscreen_message_with_choices_and_wait_P(
             lcd_show_choices_prompt_P(current_selection, first_choice, second_choice, second_col, third_choice);
         }
     }
+exit:
```

Comment:
it would be better to use some RAII pattern, but the compiler makes the jmp instructions anyway :wink: , so I don't complain